src/main/java/nguyen/Nguyen.java

```diff
@@ -0,0 +1,85 @@
+package nguyen;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+enum TaskType {
+    TODO, DEADLINE, EVENT;
+}
```

Comment:
I suggest adding a common prefix to each of these constants. 
Doing this indicates that they belong together and makes them appear together when sorted alphabetically.
```suggestion
enum TaskType {
    TASKTYPE_TODO, TASKTYPE_DEADLINE, TASKTYPE_EVENT;
}